pkg/vm/engine/disttae/txn_table.go

```diff
@@ -2627,7 +2627,7 @@ func dumpTransferMaps(ctx context.Context, taskHost *cnMergeTask) error {
 			objRowCnt++
 
 			if objRowCnt*len(columns)*int(unsafe.Sizeof(int32(0))) > 200*mpool.MB {
-				filename := blockio.EncodeTmpFileName("tmp", "merge", time.Now().UTC().Unix())
+				filename := blockio.EncodeTmpFileName("tmp", "merge", time.Now().UTC().UnixMilli())
```

Comment:
You just make the problem 1000 times harder to repro.    Here we need to use a reliable way to generate a unique name.    How about generate a uuid here?

Reply:
Added uuid to make sure unique filename.